scripts/bdairo.js

```diff
@@ -1,3 +1,10 @@
 let name= "Boluwarin Dairo";
-let id = "INTERNSHIP ID: HNG-01402"
-console.log(" Hello World, This is " + name + " with " + id +" using javascript for stage 2 task" );
\ No newline at end of file
+let id = "INTERNSHIP ID: HNG-01402";
```

Comment:
Please remove the unnecessary text in your id variable.

```suggestion
let id = "HNG-01402";
```